react/ProductCountPerPage.js

```diff
@@ -17,20 +18,40 @@ const ProductCountPerPage = () => {
     searchQuery
   )
 
+  const productsLoadedPercentage = Math.round(
+    (100 * products.length) / recordsFiltered
+  )
+
   if (products.length === 0) {
     return null
   }
 
+  const showingProducts =
+    productsLoadedPercentage === 100
+      ? {
```

Comment:
No `defineMessages` because you're using `context.json`, right?

Reply:
Added defineMessages!